README.md

```diff
@@ -1,57 +1,117 @@
+3. Under the root directory, run `make run` in your terminal
+4. Inspect the application services exposed at the following URLs.
+	- The client container exposes port 3000 locally at http://localhost:3000
+	- The server container exposes port 5000 locally at http://localhost:5000
+	- The database container exposes port 5435 and can be inspected using pgAdmin, Postico, psql, etc.
+5. Verify that the database migration container container was created and closed by running `docker ps -a | grep migrate`
+
+### OpenShift Deployment
+
+Before deploying to the BC Government OpenShift cluster, access must be granted. If you are a member of a new Developer Exchange team, you may have to request access. The BC Developer Exchange [provides a channel](https://github.com/BCDevOps/devops-requests) through which OpenShift access can be requested.
+
+1. Ensure the [OpenShift CLI](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html#cli-reference-get-started-cli) is installed `oc version`
+2. Login to your OpenShift cluster using the CLI
+	- Ensure you have access to the BC Gov OpenShift cluster (request access [here](https://github.com/BCDevOps/devops-requests/issues/new?template=openshift_user_access_request.md))
+	- Login to the BC Gov OpenShift cluster [here](https://console.pathfinder.gov.bc.ca:8443/console/)
+	- Ensure there is at least one namespace that you have rights to edit and note its ID
+	- Click your name in the top right to reveal a dropdown
+	- Click "Copy Login Command"
+	- Paste into your terminal
+3. From the root project directory, run `make create-nsp NAMESPACE=[NAMESPACE] APP_NAME=[APP_NAME]` replacing `[NAMESPACE]` with the ID of your OpenShift namespace and `[APP_NAME]` with an arbitrary application name (as per Kubernetes [requirements](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names), must meet [RFC-1123](https://tools.ietf.org/html/rfc1123))
+4. Run `make create-database NAMESPACE=[NAMESPACE] APP_NAME=[APP_NAME] POSTGRESQL_DATABASE=[POSTGRESQL_DATABASE]` replacing `[POSTGRESQL_DATABASE]` with an arbitrary database name
+5. Using the clsuter console, wait for the database to become fully deployed i.e. scale to one or more pods
+6. Run `make create-server NAMESPACE=[NAMESPACE] APP_NAME=[APP_NAME] REPO=https://github.com/bcgov/openshift-launchpad BRANCH=develop`
+7. Once the server has deployed, note the route that it exposes
+	- Using the OpenShift console, navigate to Applications > Routes
+	- The relevant route will be named [APP_NAME]-server
+	- Copy the URL that the route exposes
+7. Run `make create-client NAMESPACE=[NAMESPACE] APP_NAME=[APP_NAME] REPO=https://github.com/bcgov/openshift-launchpad BRANCH=develop API_URL=[API_URL]` replacing `[API_URL]` with the route URL copied in the previous step
+8. Once the client is built and deployed, confirm it's working by navigating to the route it exposes
```

Comment:
Local deployment was a breeze. When I get to this final step, I noticed that my server had an error. Looking at the logs, here's where it ends. Chatting with @ashtonmeuser this seems like an issue with the Network Security Policy (NSP) where it takes some time to be applied, so perhaps there's a way to know that your NSP has been successfully applied so that when a user gets to this stage, they're not stuck.

![image](https://user-images.githubusercontent.com/38572070/75193484-37c2ee00-570b-11ea-94fb-f6df0e9671cf.png)

Reply:
Made small note. Not sure what else we can do until we come up with a way to query for the status of an NSP. The `status` prop returned by `oc get nsp [YOUR NSP NAME] -o yaml` does not correctly reflect when an operator has applied the NSP.